packages/Cmfcmf-React-Fiber.package/CMFReactNodeContextProvider.extension/instance/computeChildrenAndUpdate.for.caughtError..st

```diff
@@ -0,0 +1,4 @@
+*Cmfcmf-React-Fiber
+computeChildrenAndUpdate: aFiber for: aState caughtError: anErrorOrNil
+
+	^ props at: #children
```

Comment:
Arguably it's okay like this because the constructor requires you to specify children. If you pass nil, it's also okay because we filter out `nil`s at a higher level. This passes for example:

```smalltalk
testContextWithoutChildren

	self openComponentBlock: [CMFReactTestContext asProviderNodeWithValue: 'changed' children: nil]
```